### PR TITLE
DOC-3614: move RI installation instructions under Redis OSS and Stack

### PR DESCRIPTION
**Pull request by dwdougherty**
Ticket: [DOC-3614](https://redislabs.atlassian.net/browse/DOC-3614)
Staging links:
https://redis.io/docs/staging/DOC-3614/ (home page change)
https://redis.io/docs/staging/DOC-3614/operate/oss_and_stack/install/install-redis-insight/ (Redis Insight changes)

@mich-elle-luna This is related to ticket [LRC-291](https://redislabs.atlassian.net/browse/LRC-291), in which I asked if this is the right location for these docs.

### CODE REVIEW

**Review by mich-elle-luna — APPROVED** [review]
thank you!

**Comment by dwdougherty** [comment]
@mich-elle-luna Please take one more look, especially at the home page. Thanks!

File: content/operate/oss_and_stack/install/install-redis-insight/_index.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ categories:
 - docs
 - operate
 - redisinsight
-description: 'Install RedisInsite on AWS, Docker, and Kubernetes
+description: 'Install Redis Insite on AWS, Docker, and Kubernetes
```
Comment (mich-elle-luna) [inline content/operate/oss_and_stack/install/install-redis-insight/_index.md:6]:
```suggestion
description: 'Install Redis Insight on AWS, Docker, and Kubernetes
```

File: content/operate/oss_and_stack/install/install-redis-insight/_index.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ categories:
...
 
   '
 linkTitle: Install Redis Insight
```
Comment (mich-elle-luna) [inline content/operate/oss_and_stack/install/install-redis-insight/_index.md:9]: should we now add aliases for the old location? I just thought of this

Comment (mich-elle-luna) [reply content/operate/oss_and_stack/install/install-redis-insight/_index.md:9]: should we also add aliases for the files below that are moving?

**Review by mich-elle-luna — CHANGES_REQUESTED** [review]
one typo and a couple new questions I didn't think of before

File: content/operate/oss_and_stack/install/install-redis-insight/_index.md (hunk 2 above)
Comment (dwdougherty) [reply content/operate/oss_and_stack/install/install-redis-insight/_index.md:9]: Ugh... I don't know.

Comment (dwdougherty) [reply content/operate/oss_and_stack/install/install-redis-insight/_index.md:9]: I'm going to add them in a following commit.

**Review by mich-elle-luna — APPROVED** [review]
thank you!

**Pull request merged**